tests/timeseries/_battery_pool/test_battery_pool.py

```diff
@@ -515,21 +551,43 @@ async def test_batter_pool_power_two_batteries_per_inverter(
     mockgrid = MockMicrogrid(
         graph=gen.to_graph(
             [
-                ComponentCategory.METER,
                 (
-                    ComponentCategory.INVERTER,
-                    [ComponentCategory.BATTERY, ComponentCategory.BATTERY],
+                    ComponentCategory.METER,
+                    (
+                        ComponentCategory.INVERTER,
+                        [ComponentCategory.BATTERY, ComponentCategory.BATTERY],
+                    ),
                 ),
-                ComponentCategory.METER,
                 (
-                    ComponentCategory.INVERTER,
-                    [ComponentCategory.BATTERY, ComponentCategory.BATTERY],
+                    ComponentCategory.METER,
+                    (
+                        ComponentCategory.INVERTER,
+                        [ComponentCategory.BATTERY, ComponentCategory.BATTERY],
+                    ),
```

Comment:
Inverters being connected to the main meter is a case we have to support.  If we want to remove it, it has to be discussed first.  Maybe you can bring it up on Friday.

Reply:
Oops, never mind, I see the original graph was wrong.